ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -1153,8 +1161,8 @@ def bind_ovs_ports(self, br_name, dut_iface, injected_iface, vm_iface, disconnec
                            (br_name, dut_iface_id, vm_iface_id, injected_iface_id))
             VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=5,ip,in_port=%s,action=output:%s" %
                            (br_name, dut_iface_id, injected_iface_id))
-            VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=5,ipv6,in_port=%s,action=output:%s" %
-                           (br_name, dut_iface_id, injected_iface_id))
+            VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=5,ipv6,in_port=%s,action=output:%s,%s" %
+                           (br_name, dut_iface_id, vm_iface_id, injected_iface_id))
```

Comment:
Can you submit a separate PR for this infrastructure code change made to "vm_topology.py" file?

Reply:
These changes are required for enabling PTF traffic to go through 7 node testbed. Current vm topology only assumes one direction of the traffic from other node to DUT. For 7 nodes testbed, we need to have traffic running from DUT and to DUT. That's why we need to make these changes.

Reply:
Is it a better idea to separate infrastructure change and test case change in different PRs?